options/locale/locale_en-US.ini

```
@@ -1888,7 +1888,16 @@ settings.hooks = Webhooks
 settings.githooks = Git Hooks
 settings.basic_settings = Basic Settings
 settings.mirror_settings = Mirror Settings
-settings.mirror_settings.docs = Set up your project to automatically push and/or pull changes to/from another repository. Branches, tags, and commits will be synced automatically. <a target="_blank" rel="noopener noreferrer" href="https://docs.gitea.io/en-us/repo-mirror/">How do I mirror repositories?</a>
+settings.mirror_settings.docs = Set up your project to automatically push and/or pull changes to/from another repository. Branches, tags, and commits will be synced automatically.
+settings.mirror_settings.docs.disabled_pull_mirror.instructions = Set up your project to automatically push changes to another repository. Branches, tags, and commits will be synced automatically. Pull mirrors have been disabled by your site administrator.
+settings.mirror_settings.docs.disabled_push_mirror.instructions = Set up your project to automatically pull changes from another repository. Branches, tags, and commits will be synced automatically.
+settings.mirror_settings.docs.disabled_push_mirror.pull_mirror_warning = Right now, this can only be done in the "New Migration" menu. For more information, please consult:
+settings.mirror_settings.docs.disabled_push_mirror.info = Push mirrors have been disabled by your site administrator.
+settings.mirror_settings.docs.no_new_mirrors = Your repository is mirroring changes to/from another repository. Please keep in mind that you can't create any new mirrors at this time.
```

```suggestion
settings.mirror_settings.docs.no_new_mirrors = Your repository is mirroring changes to or from another repository. Please keep in mind that you can't create any new mirrors at this time.
```

Assuming this `or` is correct here, I like it better than slash.